src/test/java/racingcar/database/RacingGameDaoTest.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.database;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.jdbc.Sql;
+
+@JdbcTest
+@Sql(scripts = {"classpath:data.sql"})
+class RacingGameDaoTest {
+    
+    private RacingGameDao gameDao;
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    
+    @BeforeEach
+    void setUp() {
+        this.gameDao = new RacingGameDao(this.jdbcTemplate);
+    }
+    
+    @Test
+    @DisplayName("insert - 게임 횟수와 우승자 이름을 받아서 DB에 저장한다.")
+    void insertGameTest() {
+        final int trialCount = 5;
+        final String winners = "io";
+        this.gameDao.insert(trialCount, winners);
+        this.gameDao.insert(trialCount, winners);
+        final String sql = "SELECT count(*) FROM racing_game WHERE trial_count = ? AND winners = ?";
+        final int count = this.jdbcTemplate.queryForObject(sql, Integer.class, trialCount, winners);
+        Assertions.assertThat(count).isEqualTo(2);
+    }
```

Comment:
아래와 같이 given when then 으로 나눠주면 테스트 가독성을 높일 수 있어요 :)
https://brunch.co.kr/@springboot/292 해당 아티클의 윗부분만 읽어보시길 추천드립니다 😄 

```suggestion
    void insertGameTest() {
        // given
        final int trialCount = 5;
        final String winners = "io";
        
        // when
        this.gameDao.insert(trialCount, winners);
        this.gameDao.insert(trialCount, winners);
        
        // then
        final String sql = "SELECT count(*) FROM racing_game WHERE trial_count = ? AND winners = ?";
        final int count = this.jdbcTemplate.queryForObject(sql, Integer.class, trialCount, winners);
        Assertions.assertThat(count).isEqualTo(2);
    }
```